WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -1089,8 +1089,13 @@ class AbstractPostListViewController: UIViewController,
         return ReachabilityUtils.noConnectionMessage()
     }
 
-    func shouldPresentAlert() -> Bool {
-        return !connectionAvailable() && !contentIsEmpty() && isViewOnScreen()
+    // MARK: - Others
+
+    override func present(_ viewControllerToPresent: UIViewController, animated flag: Bool, completion: (() -> Void)? = nil) {
+        // Override to hide Notice when we are showing the Editor. We hide Notices in
```

Comment:
I think the comment is not very clear (or at least its not to me 😬 )
I would add "This override purpose is..." or something like that.  and "Ths noticce should be dismissed/ hidden  in viewDidDisappear" 

>but it is not called when presenting a ViewController.

What is "it"?

Reply:
Thank you for letting me know. I updated it in 9bea3d94bb25fe026cc1312d7de10c5e2d91e9e4. How does that look? The mention of `viewDidDisappear` is out of context so I just removed it.